### PR TITLE
SAK-43989 add padding to grade names like A so the grade items align

### PR DESCRIPTION
**Pull request by ottenhoff**

### CODE REVIEW

File: gradebookng/tool/src/webapp/scripts/gradebook-chart.js [hunk 1]
```diff
@@ -60,15 +60,21 @@ function renderChart(gbChartData) {
...
 				yAxes: [{
 					ticks: {
 						beginAtZero:true,
 						fontStyle: 'bold',
+						fontFamily: 'Monospace',
 						autoskip: true,
-						maxRotation: 0
+						maxRotation: 0,
+						// Include a space to even out the plusses and minuses
+						callback: function(value, index, values) {
```
Comment (profmikegreene) [inline gradebookng/tool/src/webapp/scripts/gradebook-chart.js:75]: Does it matter that we use a generic callback vs one of the chartjs specific options https://www.chartjs.org/docs/latest/axes/#callbacks?

Any issue using `' '` vs `'&nbsp;'` ?

If not, looks good to me.

Comment (ottenhoff) [reply gradebookng/tool/src/webapp/scripts/gradebook-chart.js:75]: my guess is nbsp would be escaped... ' ' works as expected here

Comment (ottenhoff) [reply gradebookng/tool/src/webapp/scripts/gradebook-chart.js:75]: screenshot in the JIRA

**Review by adrianfish — APPROVED** [review]
GTG. Monospaced font to ensure that the alignment and length checks work reliably. Makes sense.

**Pull request merged**